org.eclipse.dartboard/src/org/eclipse/dartboard/propertytester/IsDartProjectPropertyTester.java

```diff
@@ -38,7 +39,7 @@ public boolean test(Object receiver, String property, Object[] args, Object expe
 
 			try {
 				for (IResource res : project.members()) {
-					if ("pubspec.yaml".equals(res.getName()) || "dart".equals(res.getFileExtension())) { //$NON-NLS-1$ //$NON-NLS-2$
+					if (Constants.PUBSPEC.equals(res.getName()) || "dart".equals(res.getFileExtension())) { //$NON-NLS-1$
```

Comment:
Since we know the default path. We can check in the default path using	`#findMember(String)`
If not found,  we need to iterate all the members.

Reply:
Done